polyfill/intersection-observer.js

```diff
@@ -62,7 +62,7 @@ function IntersectionObserverEntry(entry) {
 
   // Sets intersection ratio.
   if (targetArea) {
-    this.intersectionRatio = intersectionArea / targetArea;
+    this.intersectionRatio = (intersectionArea / targetArea).toFixed(3) - 0;
```

Comment:
Can you use either `Number()` or `parseFloat()` here instead of `- 0` to do the type conversion? I understand your way is shorter, but it's less clear why it's happening.